core/bcast/errors_rate.go

```diff
@@ -0,0 +1,95 @@
+// Copyright © 2022-2024 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package bcast
+
+import (
+	"sync/atomic"
+
+	"github.com/obolnetwork/charon/core"
+)
+
+const (
+	// RegistrationErrorsRateThreshold defines the threshold for registration errors rate.
+	RegistrationErrorsRateThreshold int32 = 70
+)
+
+// errorsRate object tracks registration errors rate within the last two epochs and updates metrics.
+type errorsRate struct {
```

Comment:
```suggestion
type builderRegistrationErrorsRate struct {
```

Not the best name ever, but errorsRate seems a little too generic.